e2e/mock-data.ts

```diff
@@ -0,0 +1,84 @@
+import { readFileSync } from "fs";
+import path from "path";
+import { XMLParser } from "fast-xml-parser";
+
+const parser = new XMLParser();
+const dimensionDoc = parser.parse(
+  readFileSync(path.resolve(__dirname, "../nv-mock/data/dimensionlist.nv.xml")),
```

Comment:
Just a thought, no need to change anything at this point. In general, I feel we should not have to refer to files outside of the current NPM project, especially with `path`.

1. Simple: Maybe `nv-mock` should just live inside of `e2e/`. I chose to put it at the repo root not seeing the quite clear connection between...well, tests and test data... (However, *do not* move it in this PR. The diff is already large enough.)
2. Complex: Not sure but I think Mockoon supports template files, i.e., we could have the actual test/mock data as `.json` or `.ts` and XML would be used in templates only. If sufficient features are available, this would probably be the idiomatic way but it also feels like it might be a little too much work at this point. Maybe something to keep in mind.